### PR TITLE
Add The Quacks of Quedlinburg

### PR DESCRIPTION
**Pull request by suravshrestha**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -253,6 +253,20 @@
...
+> 
+> Each player has their own bag of ingredient chips. During each round, they simultaneously draw chips from their bags and add them to their pots. The higher the face value of the drawn chip, the further it is placed in the pot's swirling pattern, increasing how much the potion will be worth. Push your luck as far as you can, but if you add too many cherry bombs, your pot will explode!
+> 
+> At the end of each round, players gain victory points and coins to spend on new ingredients, depending on how well they managed to fill up their pots. But players whose pots have exploded must choose points or coins — not both! The player with the most victory points at the end of nine rounds wins the game.
+
+![The Quacks of Quedlinburg game image](https://cf.geekdo-images.com/pH5LgRL4mNRon-2NsSDb1Q__itemrep/img/qe9ywOTxtPr_n4DoXZgEY-C2qU0=/fit-in/246x300/filters:strip_icc()/pic6137509.png)
+
+| Players | Min. Age | Time   |
+| ------: | -------: | -----: |
+|   2 - 5 |        8 | 30-60m |
```
Comment (edm00se) [inline readme.md:268]:
```suggestion
|   2 - 4 |       10 |  45m |
```

Please correct the stats blcok.

**Review by edm00se — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (suravshrestha) [reply readme.md]: @edm00se, I've implemented the required modifications. Would you please review the PR and proceed with the merge? Thank you!

Comment (suravshrestha) [reply readme.md]: > Please correct the stats block. We seek to be in agreement with Board Game Geek's listing (which is usually accurate to the publisher's).

@edm00se, I've implemented the required modifications. Would you please review the PR and proceed with the merge? Thank you!

**Review by edm00se — APPROVED** [review]
That did the trick, thanks for contributing.

refs: #269

**Pull request merged**